content/en/docs/contribution-guidelines/upgrade-graphs.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Upgrade Graphs Diagrams
+linkTitle: Upgrade Graphs
+---
+
+## Standards and definitions
+
+| <div style="align-content: center;">Description</div>   | <div style="align-content: center;width:200px">Diagram Example</div>       | 
+|----------|:-------------:|
+| bundle which is the head of the channel | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1):::head
+{{</mermaid>}} |
+| bundle which is the installed currently | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1):::installed
+{{</mermaid>}} |
+| bundle which can be installed | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1)
+{{</mermaid>}} |
+| channel | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   subgraph preview
+      ID(v0.0.1) 
+   end
+{{</mermaid>}} |
+| default channel | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   subgraph "preview (default)"
+      ID(v0.0.1) --> ID2(v0.0.2)
+   end
+{{</mermaid>}} |
+| replaces action to specify an upgrade path for a project which will replace one version for another. It is defined in the CSV. (eg.`spec.replaces: exampleoperator.v0.0.1`) | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   A(v0.0.1) --> B(v0.0.4)
+{{</mermaid>}} |
+| skips action to skip one or more specific project versions (1..N). It is defined in the CSV (eg.`spec.skips: exampleoperator.v0.0.2, exampleoperator.v0.0.3`)| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   A(v0.0.1) x--x |v0.0.2,v0.0.3| B(v0.0.4)
+{{</mermaid>}} |
+| skipRange action which specify a range that should not be used to upgrade a specific version to another. It is defined in the CSV. (E.g `annotations.olm.skipRange: '>= 2.0.4 < 3.0.0'`) | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   C(v2.0.3) o--o |>= 2.0.4 < 3.0.0| D(v3.0.1)
+{{</mermaid>}} |
+| future replace | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   A(v0.0.1) -.-> B(v0.0.4)
+{{</mermaid>}} |
+| future skips | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   A(v0.0.1) x-.-x |v0.0.2,v0.0.3| B(v0.0.4)
+{{</mermaid>}} |
+| future skipRange | {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   C(v2.0.3) o-.-o |>= 2.0.4 < 3.0.0| D(v3.0.1)
+{{</mermaid>}} |
+| OCP label which is defined in the bundle Dockerfile image to define which OCP version(s) the project will be supported. (eg. `LABEL com.redhat.openshift.versions:=v4.7`)|  {{<mermaid>}}
```

Comment:
We are actually avoiding mention any red hat specific stuff here (OCP, redhat registry etc) since this is a strictly upstream doc site. So we'll have to use example labels instead.